test/TravelersRepository-test.js

```diff
@@ -0,0 +1,36 @@
+import chai from 'chai';
+const expect = chai.expect;
+import TravelersRepository from '../src/TravelersRepository.js';
+import travelersSampleData from '../data/travelers.js';
+
+describe('TravelersRepository', function() {
+
+  let travelersRepository;
+
+  beforeEach(() => {
+    travelersRepository = new TravelersRepository(travelersSampleData);
+  })
+
+  it('Should be a function', function() {
+    expect(TravelersRepository).to.be.a('function');
+  });
+
+  it('should be an instance of UserRepository', () => {
```

Comment:
I think you meant TravelersRepository here on line 18 instead of UserRepository :)

Reply:
Such a good catch!! I will change this ASAP!